main.go

```diff
@@ -54,7 +58,8 @@ func main() {
 	}
 
 	ctx := context.Background()
-	dumper, err := NewDumper(ctx, opts.ProjectId, opts.InstanceId, opts.DatabaseId, os.Stdout, timestamp, opts.BulkSize)
+	tables := strings.Split(opts.Tables, ",")
```

Comment:
If opts.Tables is empty string, `strings.Split(opts.Tables, ",")` produces a slice which has a single value `""`.
https://play.golang.org/p/gaOXKS58kw0

This should be an empty slice, otherwise spanner-dump can't work without `--tables` option.

Reply:
you are right, let me change it